jupyter_server_ydoc/handlers.py

```diff
@@ -37,8 +38,8 @@ class DocumentRoom(YRoom):
 
     is_transient = False
 
-    def __init__(self, type: str, ystore: BaseYStore):
-        super().__init__(ready=False, ystore=ystore)
+    def __init__(self, type: str, ystore: BaseYStore, log: Logger):
```

Comment:
```suggestion
    def __init__(self, type: str, ystore: BaseYStore, log: Optional[Logger]=None):
```

Should this be optional too as it is in the base class?